core/src/main/java/com/linecorp/armeria/client/cookie/DefaultCookieJar.java

```diff
@@ -56,7 +59,7 @@
 
     DefaultCookieJar(CookiePolicy cookiePolicy) {
         this.cookiePolicy = cookiePolicy;
-        store = new HashMap<>();
+        store = new Object2LongOpenHashMap<>();
```

Comment:
You have to change the type of `store` to `Object2LongMap` and use the methods that returns and accepts `long` instead of `Long` to take full advantage of this.

Reply:
Ya sorry this is new to me 😆